src/main/java/com/microsoft/sqlserver/jdbc/ISQLServerBulkData.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Microsoft JDBC Driver for SQL Server Copyright(c) Microsoft Corporation All rights reserved. This program is made
+ * available under the terms of the MIT License. See the LICENSE file in the project root for more information.
+ */
+
+package com.microsoft.sqlserver.jdbc;
+
+import java.io.Serializable;
+
+/**
+ * Provides an interface used to create classes that read in data from any source (such as a file) and allows a
+ * SQLServerBulkCopy class to write the data to SQL Server tables.
+ */
+public interface ISQLServerBulkData extends Serializable {
+
+    /**
+     * Returns the ordinals for each of the columns represented in this data record.
+     *
+     * @return Set of ordinals for the columns.
+     */
+    java.util.Set<Integer> getColumnOrdinals();

```

Comment:
I know that public modifiers are redundant for interfaces, but could we add the public modifiers to the methods in this interface, just for the sake of consistency with other interfaces? Either that or please remove the public modifiers in the other interfaces as well (or you can let us handle this in the near future)

Reply:
I've removed from all your interfaces